aicsimageio/readers/default_reader.py

```diff
@@ -256,13 +256,6 @@ def _unpack_dims_and_coords(
         # Use dims for coord determination
         coords = {}
 
-        # Handle typical RGB and RGBA from channels
-        if DimensionNames.Channel in dims:
-            if image_data.shape[dims.index(DimensionNames.Channel)] == 3:
-                coords[DimensionNames.Channel] = ["R", "G", "B"]
-            elif image_data.shape[dims.index(DimensionNames.Channel)] == 4:
-                coords[DimensionNames.Channel] = ["R", "G", "B", "A"]
```

Comment:
rather than delete, these should be setting coords[DimensionNames.Sample], right?

Reply:
:facepalm: _Great_ recommendation.

Reply:
I was only thinking about the `.channel_names` property when I did this.